modules/ossm-applying-certificates-to-multi-cluster-topology.adoc

```diff
@@ -0,0 +1,114 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-applying-certificates-to-multi-cluster-topology_{context}"]
+= Applying certificates to a multi-cluster topology 
+
+This procedure explains how to apply security certificates to a multi-cluster topology.
+
+.Prerequisites
+
+* You have access to two {ocp-product-title} clusters with external load balancer support. 
+
+* You have installed the {SMProduct} 3.0 Operator to two clusters.
+
+* You have created the root CA certificate and intermediate CA certificates for each cluster or someone has made them available for you. 
+
+[NOTE]
+====
+In this procedure, `cluster1` is the East cluster and `cluster2` is the West cluster. 
+====
+
+.Procedure
+
+. Log in to the first cluster by running the following command:
++
+[source,terminal]
+----
+$ $ oc login -u https://<first_cluster_api_server_url>
+----
+
+. Set up the environment variable that contains the context for the first cluster by running the following command:
++
+[source,terminal]
+----
+$ export CTX_CLUSTER1=$(oc config current-context)
+----
+
+. Create a namespace called `istio-system` by running the following command:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER1}" || oc create namespace istio-system --context "${CTX_CLUSTER1}"
+----
+
+. Configure {istio} to use `network1` as the default network for the pods on the East cluster by running the following command:
++
+[source,terminal]
+----
+oc --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+----
+
+. Create the CA certificate secret for {istio} on the East cluster by running the following command:
++
+[source,terminal]
+----
+oc get secret -n istio-system --context "${CTX_CLUSTER1}" cacerts || oc create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+  --from-file=east/ca-cert.pem \
+  --from-file=east/ca-key.pem \
+  --from-file=east/root-cert.pem \
+  --from-file=east/cert-chain.pem
+----
++
+[NOTE]
+====
+If you followed the instructions in _Creating certificates for a multi-cluster mesh_ then your certificates will reside in the directory specified in the previous example. If your certificates reside in a different directory, modify the syntax accordingly. 
+====
+
```

Comment:
This is a good place to split the procedure into two sections, so you'll have two sections of 5 steps each instead of one section with 10 steps. The steps for cluster1 end here; steps for cluster2 follow from here till the end of the document.

Reply:
done, but let's look at this and discus it.

Reply:
done